### DISCUSSION TITLE
Inaccessible Bootstrap CDN Link - Privacy Error (?)

### DISCUSSION THREAD

**Discussion by janpul** [twbs/bootstrap#40268] [Q&A] [answered]
### Prerequisites

- [X] I have [searched](https://github.com/twbs/bootstrap/issues?utf8=%E2%9C%93&q=is%3Aissue) for duplicate or closed issues
- [X] I have [validated](https://html5.validator.nu/) any HTML to avoid common problems
- [X] I have read the [contributing guidelines](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md)

### Describe the issue

It seems that the link to Bootstrap is being alerted with a Privacy Error, making the file inaccessible on my side. I asked my colleagues regarding this issue, and they said that it is working on their side. What can I do to fix this issue?

Privacy Error Screen:
![isntprivate](https://github.com/twbs/bootstrap/assets/76933383/5716c140-f65c-4d6e-a525-6b36c607dac0)

Not Found:
![image](https://github.com/twbs/bootstrap/assets/76933383/8020cb8d-e830-4d22-8969-d133d46393d4)

### Reduced test cases

N/A

### What operating system(s) are you seeing the problem on?

Windows

### What browser(s) are you seeing the problem on?

Microsoft Edge

### What version of Bootstrap are you using?

v4.5.3 / v.5.3.3

**Comment by julien-deramond** [comment, marked as answer]
Hello @janpul 
I cannot really say, we're not maintaining jsDelivr.

> I asked my colleagues regarding this issue, and they said that it is working on their side.

Are they in the same country as you? We know that jsDelivr can be blocked in certain countries.
In this case, you might want to switch to [alternative CDNs](https://getbootstrap.com/docs/5.3/getting-started/download/#alternative-cdns).

Otherwise, it might be a network specific configuration on your side.

**Reply by janpul** [reply]
I think this is an issue with my ISP, so I might look for alternative CDNs if this is the case. Thanks for the response!
